includes/gateways/stripe/includes/class-give-stripe-webhooks.php

```diff
@@ -63,16 +63,23 @@ public function listen() {
 				return;
 			}
 
+			// Retrieve the request's body and parse it as JSON.
+			$body  = @file_get_contents( 'php://input' );
```

Comment:
Silencing all errors isn't the best idea, let's change this to a try & catch so we can process the errors and actually surface something that is wrong or bail out.